docs/ocis/development/testing.md

```diff
@@ -484,6 +498,9 @@ ocis/bin/ocis server
 ```
 {{< hint info >}}
 The value for `ANTIVIRUS_CLAMAV_SOCKET` is an example which needs adaption according your OS. If you are running `clamAv` with docker as per this documentation check the path that you mounted the volume i.e. `/your/local/filesystem/path/` to make sure the socket exists and give the full path to socket i.e. `/your/local/filesystem/path/clamd.sock` to `ANTIVIRUS_CLAMAV_SOCKET`.
+
+For antivirus running localy on Mac OS, use `ANTIVIRUS_CLAMAV_SOCKET= "/tmp/clamd.socket"`.
+For antivirus running with docker on Mac OS, use `ANTIVIRUS_CLAMAV_SOCKET= "tcp://localhost:3310"`.
```

Comment:
I didn't know we could give the value like this maybe we can do the same for linux too

Reply:
Also you might not need volumes if we cab directly communicate through tcp

Reply:
without volumes I catch ocis crash: 
```
{"level":"error","service":"gateway","pkg":"rgrpc","traceid":"00000000000000000000000000000000","error":"internal error: gateway: error finding an auth provider for type: bearer","type":"bearer","time":"2023-06-21T12:47:06.690014+02:00","message":"error getting auth provider client"}
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x48 pc=0x103244760]

goroutine 163 [running]:
github.com/owncloud/ocis/v2/services/antivirus/pkg/scanners.ClamAV.Scan({0x14001d51718?}, {0x132e0b458, 0x14000d481b0})
        /Users/scharfviktor/Work/ocis/services/antivirus/pkg/scanners/clamav.go:31 +0x80
github.com/owncloud/ocis/v2/services/antivirus/pkg/service.Antivirus.process({0x140017fd440, {{{0x105e68b98, 0x1400163bd30}, 0x3, {0x0, 0x0}, {0x14001998200, 0x16, 0x1f4}, {0x1400163bd40, ...}, ...}}, ...}, ...)
        /Users/scharfviktor/Work/ocis/services/antivirus/pkg/service/service.go:192 +0x2f8
github.com/owncloud/ocis/v2/services/antivirus/pkg/service.Antivirus.Run({0x140017fd440, {{{0x105e68b98, 0x1400163bd30}, 0x3, {0x0, 0x0}, {0x14001998200, 0x16, 0x1f4}, {0x1400163bd40, ...}, ...}}, ...})
        /Users/scharfviktor/Work/ocis/services/antivirus/pkg/service/service.go:129 +0x560
github.com/oklog/run.(*Group).Run.func1({0x1400199c0a0?, 0x1400163bd70?})
        /Users/scharfviktor/Work/ocis/vendor/github.com/oklog/run/group.go:38 +0x34
created by github.com/oklog/run.(*Group).Run
        /Users/scharfviktor/Work/ocis/vendor/github.com/oklog/run/group.go:37 +0x68
        
        ```